app/models/organization.rb

```diff
@@ -1,4 +1,12 @@
 class Organization < ApplicationRecord
   has_many :user_organizations
   has_many :users, through: :user_organizations
+
+  validates :name,
```

Comment:
Merge these two validations into one. Add validation for presence of relationship objects.